### PR TITLE
fix(onboarding-1.0): blue buttons and small tweaks

### PR DESCRIPTION
**Pull request by liyiy**
## Changes

Screen Shot 2022-05-02 at 11 24 30 PM
Screen Shot 2022-05-02 at 11 24 24 PM
Screen Shot 2022-05-02 at 11 25 53 PM

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

**Review by clarkus — COMMENTED** [review]
How does the breadcrumbs / step navigation work? I see there's an additional back arrow, but the idea was that the steps could also be linked much like breadcrumbs?

**Comment by liyiy** [comment]
> How does the breadcrumbs / step navigation work? I see there's an additional back arrow, but the idea was that the steps could also be linked much like breadcrumbs?

We can switch to step navigation instead. Initially I was still waiting for feedback on that so I left the original back arrow in there. It should be done in another PR instead though. I will work on that after I get third party integration merged.

**Review by rcmarron — APPROVED** [review]
LGTM! Just made one little padding tweak to the top of the skip button. Also, I think once my other PR is in we might want to move these buttons to the `size="large"`. But we can handle that then.

**Pull request merged**
